Project/Modules/Controllers/PostsController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using blog_net_core.Project.Modules.Model.Entities;
+using blog_net_core.Project.Modules.Model.Repositories;
+using blog_net_core.Project.Modules.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace blog_net_core.Controllers
+{
+    [Route("[controller]")]
```

Comment:
can you explain me why you use the [Route("[controller]")] and which is the difference with [Route("posts")]

Reply:
there is no difference between [Route ("[controller]")] and [Route ("posts")]. Corrected